README.md

```diff
@@ -1,2 +1,174 @@
+
+***
+#### Reproduce results for simulation II
+* simulated data with 100 total samples and 0 % missing values (N100_M0)
+
+R code for simulation: simulate_N100_M0.R;
+
+R code for applying Bayesian SFPCA: sfpca_N100_M0.R;
+
+Figure 4a (Mean_trueVsestimated.pdf); 
+
+Figure 4b (FPCs_trueVsestimated.pdf);
+
+Figure 4c (fpcScores_scatterplots.pdf)
+
+* simulated data with 100 total samples and 80 % missing values (N100_M80)
+
+R code for simulation: simulate_N100_M80.R;
+
+R code for using splinectomeR: figure_splinectome_N100_M80.R; 
+
+R code for applying Bayesian SFPCA: sfpca_N100_M80.R;
+
+Figure 4d (Mean_trueVsestimated.pdf); 
+
+Figure 4e (FPCs_trueVsestimated.pdf);
+
+Figure 4f (fpcScores_scatterplots.pdf) 
+
+* simulated data with 50 total samples and 80 % missing values (N50_M80)
+
+R code for simulation: simulate_N50_M80.R;
+
+R code for applying Bayesian SFPCA: sfpca_N50_M80.R;
+
+Figure 5a (Mean_trueVsestimated.pdf);
+
+Figure 5b (FPCs_trueVsestimated.pdf);
+
+Figure 5c (fpcScores_scatterplots.pdf)
+
+* simulated data with 25 total samples and 80 % missing values (N25_M80)
+
+R code for simulation: simulate_N25_M80.R;
+
+R code for using splinectomeR: figure_splinectome_N25_M80.R;
+
+R code for applying Bayesian SFPCA: sfpca_N25_M80.R;
+
+Figure 5d (Mean_trueVsestimated.pdf);
+
+Figure 5e (FPCs_trueVsestimated.pdf);
+
+Figure 5f (fpcScores_scatterplots.pdf)
+
+* simulated data with 10 total samples and 50 % missing values (N10_M50)
+
+R code for simulation: simulate_N10_M50.R;
+
+R code for applying Bayesian SFPCA: sfpca_N10_M50.R;
+
+Figure 6a (Mean_trueVsestimated.pdf);
+
+Figure 6b (FPCs_trueVsestimated.pdf);
+
+Figure 6c (fpcScores_scatterplots.pdf)
+
+* simulated data with 10 total samples and 80 % missing values (N10_M80)
+
+R code for simulation: simulate_N10_M80.R;
+
+R code for using splinectomeR: figure_splinectome_N10_M80.R;
+
+R code for applying Bayesian SFPCA: sfpca_N10_M80.R;
+
+Figure 6d (Mean_trueVsestimated.pdf);
+
+Figure 6e (FPCs_trueVsestimated.pdf);
+
+Figure 6f (fpcScores_scatterplots.pdf)
+
+***
+#### Reproduce results for ECAM data application
+* Observed data (Shannon_modelSelection.R)
+
+Figure 7a: Observed_spaghetti.pdf
+
+Figure 7b: Observed_shannon_abx.pdf
+
+Figure 7c: Observed_shannon_delivery.pdf
+
+Figure 7d: Observed_shannon_diet.pdf
+
+* Results from splinectomeR (figure_splinectome.R)
+
+Figure 7e: splinectome_delivery_curves.pdf
+
+Figure 7f: splinectome_diet_curves.pdf
+
+* Results from Bayesian SFPCA (model selection: Shannon_modelSelection.R + model fitting: sn_3pc_1n.R)
+
+Figure 8a: mean_spaghetti_transform.pdf
+
+Figure 8b: FPCs_mean_PC1.pdf
+
+Figure 8c: FPCs_mean_PC2.pdf
+
+Figure 8d: FPCs_mean_PC3.pdf
+
+Figure 9a: boxplot_abx_PC3_scores.pdf
+
+Figure 9b: boxplot_delivery_PC1_scores.pdf
+
+Figure 9c: boxplot_delivery_PC2_scores.pdf
+
+Figure 9d: boxplot_diet_PC1_scores.pdf
+
+Figure 9e: boxplot_diet_PC2_scores.pdf
+
+Figure 9f: boxplot_diet_PC3_scores.pdf
+
```

Comment:
Can you remove these last few empty lines?

Reply:
Just did. Thanks!